### DISCUSSION TITLE
Blender from steam always start with zluda?

### DISCUSSION THREAD

**Discussion by denzellejohn120** [vosen/ZLUDA#200] [Q&A]
Can the blender from steam always start with zluda without needing to type in command prompt?

**Comment by californiasunset** [comment]
> Can the blender from steam always start with zluda without needing to type in command prompt?

I think no. I myself use Blender from Steam, and the only thing is to launch it through Zluda manually. But the idea is good.

**Reply by denzellejohn120** [reply]
I found a way by making a bat file with the commands needed to start Blender with zluda.
